tools/interfaces.md

```diff
@@ -1,5 +1,69 @@
-# Work in Progress
+Mozilla Firefox Data Analysis Tools
+-----------------------------------
 
-This document is a work in progress.
-The work is being tracked in
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341896).
+This is a starting point for making sense of (and gaining access to) all of the
+Firefox-related data analysis tools. There are a number of different tools
+available, all with their own strengths, tailored to a variety of use cases and
+skill sets.
+
+####[sql.telemetry.mozilla.org](stmo.md) (STMO)
```

Comment:
Can you follow the opening `#` sequence with a space (here and throughout)? The article renders well in gitbook as-is, but does not render as a header in github.